src/SprykerEco/Zed/Heidelpay/Persistence/HeidelpayQueryContainer.php

```diff
@@ -154,4 +154,19 @@ public function queryRegistrationByIdAndQuoteHash(int $idRegistration, string $q
             ->filterByIdCreditCardRegistration($idRegistration)
             ->filterByQuoteHash($quoteHash);
     }
+
+    /**
+     * {@inheritDoc}
+     *
+     * @api
+     *
+     * @param int $idSalesOrder
+     *
+     * @return \Orm\Zed\Heidelpay\Persistence\SpyPaymentHeidelpayTransactionLogQuery
+     */
+    public function queryCaptureSuccessTransactionLog(int $idSalesOrder): SpyPaymentHeidelpayTransactionLogQuery
```

Comment:
Create a repository method instead, we are not creating query container methods.